### DISCUSSION TITLE
文档翻译增加 选择翻译引擎的设置 可以自行选择翻译api

### DISCUSSION THREAD

**Discussion by wzgrx** [immersive-translate/immersive-translate#1726] [Ideas]
### 新功能 | New Feature

比如翻译txt想使用阿里云api和火山api 对比，没有找到设置
![Snipaste_2024-05-19_23-39-40](https://github.com/immersive-translate/immersive-translate/assets/39661556/42cc7e0b-f181-4f69-82dd-93f567e3a62d)
这个界面加个选择，因为只有翻译文档才需要api，平常翻译网页用微软免费的就行，

### 为什么您想要添加这些功能 | Why you want to add these features

这个界面加个选择，因为只有翻译文档才需要api，平常翻译网页用微软免费的就行，

**Comment by theowenyoung** [comment]
直接在当前页面，点击 插件面板，切换翻译服务即可。

火山和腾讯翻译 需要用户自己配置API后才会出现。

**Comment by wzgrx** [comment]
> 直接在当前页面，点击 插件面板，切换翻译服务即可。
> 
> 火山和腾讯翻译 需要用户自己配置API后才会出现。

好的，谢谢
